assets/js/workflow-diagram/usePlaceholders.ts

```diff
@@ -51,17 +50,15 @@ export const create = (parentNode: Flow.Node) => {
 
 export default (
   el: HTMLElement | null | undefined,
-  store: StoreApi<WorkflowState>,
   requestSelectionChange: (id: string | null) => void // TODO more like changeSelection
 ) => {
+  const { add: addTo } = useWorkflowStore();
```

Comment:
what about:
```
const store = useWorkflowStore()
store.add(toWorkflow(placeholders))
```

Reply:
Yeah makes sense. I think the right pattern is to even also use selectors.